### ISSUE TITLE
BUG ON RAID DRAGON SOUL - ALERT PRIORITY - Aggro radius

### ISSUE THREAD

**Issue by Jean1noob** [gamefreedomgit/Maelstrom#6166]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/cata)

**Description:**
Each boss in the game has an intended interaction radius with players, typically designed so that the boss engages when players enter this range. However, there is a bug where the server ignores this interaction radius. As a result, bosses only interact with players when certain actions occur, such as a spell being cast or when the tank physically touches or attacks the boss. This causes the boss to remain passive even if players are within its engagement range, until direct interaction via abilities or touch triggers the encounter.
Examplo:

There are several instances in Dragon Soul where bosses fail to initiate combat because their interaction radius is not functioning as intended. Here are a few examples:

Hagara the Stormbinder: Despite standing directly next to her, Hagara will not engage with players unless a spell is cast or the tank initiates contact. Players can approach or even circle her without triggering combat.

Warmaster Blackhorn: After Warmaster Blackhorn descends onto the ship, he will not engage in combat immediately. Even at point-blank range, with players standing directly in front of him, he won't react unless a spell is cast or the tank touches him.

Morchok: This bug is particularly evident with Morchok. Players can walk directly through him without triggering the encounter. You can pass through his model repeatedly or even cross his aggro range without him attacking. This allows players to completely bypass Morchok and proceed straight to the Temple of Dragon Soul without engaging him at all.

This issue affects all bosses in Dragon Soul, preventing proper engagement when players enter their aggro radius. The bosses will only react when there's a direct action, such as casting a spell or initiating physical contact, which breaks the intended mechanics of the instance.

**How to reproduce:**
Approach any boss in the game, staying within the intended interaction radius without using any abilities or spells.
Observe that the boss does not react or engage, even though you are within the range that should trigger combat.

**How it should work:**
When a player enters the boss's intended interaction radius, the boss should immediately engage in combat, regardless of whether any spells are cast or abilities used. This means:

As soon as any player crosses into the boss's aggro range, the boss should automatically target and begin interacting with the player, following its normal threat and combat mechanics.
No direct spell cast, ability use, or physical contact (like auto-attacks) should be required for the boss to engage. Proximity alone should trigger the encounter, as designed.
This ensures that bosses behave consistently with their aggro mechanics and initiate combat based solely on players entering their interaction range.

**Database links:**
This exemplo shows players keeping distance to Hagara before the beginning
https://www.youtube.com/watch?v=RxChXHBVbwU

This is particularly common in instances, where engaging a boss typically aggro's any remaining
https://wowpedia.fandom.com/wiki/Aggro_radius

**Comment by jeanpyere** [comment]
I don't think this is an exploit or bug abuse, it is necessary to check if it works like this on Blizzard

**Comment by Jean1noob** [comment]
It’s understandable to question whether this issue is a true bug or just the intended mechanics of the encounter. However, bosses are typically designed to engage when players enter their aggro radius, so the fact that bosses in Dragon Soul do not react until a spell is cast or the tank initiates contact suggests that this behavior is unintended. To confirm if this is a bug specific to certain private servers or if it occurs on Blizzard’s servers as well, it would be necessary to compare the behavior in an official World of Warcraft environment.

**Comment by Jean1noob** [comment]
This behavior can easily be checked by watching any Dragon Soul encounter video on YouTube. In these videos, you’ll consistently see that players maintain a careful distance from bosses before pulling, making sure not to enter the boss's aggro range until they are ready to start the fight. This demonstrates that on official Blizzard servers, bosses do engage when players enter their proximity, unlike the issue where bosses remain passive until a spell or physical contact is made. This comparison further supports that the interaction radius is intended to trigger combat immediately.

**Comment by Sketish** [comment]
Morchok: This bug is particularly evident with Morchok. Players can walk directly through him without triggering the encounter. You can pass through his model repeatedly or even cross his aggro range without him attacking. This allows players to completely bypass Morchok and proceed straight to the Temple of Dragon Soul without engaging him at all.

But drakes shouldnt be available untill you kill Morchok. If that's the case then it will be fixed,

**Comment by Sketish** [comment]
Checking on ptr and cant use drakes before killing Morchok so its not abuseable. And yes some bosses are not pulling unless you hit them and some have aggro radius to pull

**Comment by Jean1noob** [comment]
No Bosses in DS has aggro radius.

**Comment by Jean1noob** [comment]
Here’s a list of all the bosses in Dragon Soul, along with the issue where none of them have an aggro radius:

Morchok: You can walk right between Morchok’s legs without him engaging. In fact, you can completely bypass him and head straight to the Temple of Dragon Soul. He won’t attack, even if you're right in front of him.

Warlord Zon'ozz: Like other bosses, you can stand right next to him, jump around, or even dance in front of him, and he won’t react until he is physically hit or a spell is cast.

Yor’sahj the Unsleeping: Yor’sahj also lacks an aggro radius. Players can walk right up to him, and unless they attack, he won’t initiate combat.

Hagara the Stormbinder: You can be directly beside Hagara without her engaging in combat. She only reacts when she is attacked or touched by a spell.

Ultraxion: Despite being a huge dragon, Ultraxion won’t engage when players approach him. You can get very close without triggering the encounter until you perform an action.

Warmaster Blackhorn: After Blackhorn descends onto the ship, you can stand right in front of him without him engaging, unless you hit him first.

Spine of Deathwing: The encounter mechanics don’t begin until certain conditions are met, but Deathwing won’t react immediately to player presence on his back.

Madness of Deathwing: As with the Spine, Deathwing doesn't react based on proximity alone. Becouse you must talk to NPC to start, But you can jump to other platform and hit TOYs prepering yourself bursting after enconter. Ex: When you jump to other platform you can wait your mana be back if noone hit boss.

**Comment by Sketish** [comment]
https://youtu.be/mSs7lK4ersI?t=17 
as example boss is not pulling by being close to him so it;s not a bug
https://youtu.be/6soOkMgoZeM?t=4
https://youtu.be/5S7RI2EeHxY

So this issue only  matters on 3 last bosses

**Comment by Jean1noob** [comment]
Follow attached people keeping distance from bosses, Becouse AGGRO RADIUS, and they started after hit agro radio:
This VIDEOS have 12 yrs, it is original.

Yor’sahj the Unsleeping: https://www.youtube.com/watch?v=-ju0GxmNRCk&rco=1

Hagara the Stormbinder: https://www.youtube.com/watch?v=RxChXHBVbwU

Morchok: https://www.youtube.com/watch?v=JOB0qZQxTOM

**Comment by Jean1noob** [comment]
Another thing is that you can literally spit on the bosses and nothing will happen. To use the "/spit" emote on someone, you need to be within a minimum distance, yet none of the bosses have an aggro radius, so you can do whatever you want without triggering any interaction. There is no minimal engagement with the game beyond the mechanics themselves. Normally, bosses should react to players entering their aggro range, but in Dragon Soul, you can perform emotes like spitting, dancing, or jumping right in front of them without any response.

**Comment by Jean1noob** [comment]
There is no human or NPC interaction at all—only the programmed mechanics function as intended. Because there is no aggro radius, the bosses don't react to players in any way outside of the specific encounter mechanics. You can approach them, perform emotes, or even stand right next to them, and they will remain completely unresponsive. This lack of aggro radius removes any dynamic interaction, leaving only the strict scripted mechanics to trigger combat.

**Comment by Jean1noob** [comment]
There’s no sense that the monster is truly alive or interacting with you. There’s no feeling of fear or tension when standing near the boss. It’s just a machine, one that will only start its mechanics when someone presses the button, so to speak. The lack of an aggro radius removes any immersive feeling that the boss is aware of your presence, turning it into a purely scripted event with no natural engagement.

**Comment by Sketish** [comment]
There is difference between sugesstion and bug report. I sent you retail proofs that those bosses didnt had aggro range. It will be changed for 3 last bosses.

**Comment by Yarikels666** [comment]
That is the goal of this report is to create chaos in pug raids by adding agro radius for bosses?

**Comment by jeanpyere** [comment]
all npcs is dead b4 morchok down, after start cinematic for ress hims
